Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/DocumentPropertyEditor.cpp

```diff
@@ -768,6 +776,49 @@ namespace AzToolsFramework
         return lastDescendant;
     }
 
+    AZ::Dom::Path DPERowWidget::BuildDomPath()
```

Comment:
On the fence if this is fine or a static factory-like function that takes a DPERowWidget* and returns an AZ::Dom::Path would be better.